site/docs/v1/tech/shared/_deleting-your-account.adoc

```diff
@@ -0,0 +1,25 @@
+To delete your account, take the following steps, in this order:
+
+1. https://account.fusionauth.io/[Log into your account].
+2. Delete all deployments.
+3. Switch your account to the community plan.
+4. Delete your company.
+
+[NOTE.note]
+====
+FusionAuth requires that customers delete their own deployments using the account portal.
+
+FusionAuth is **not able** to cancel accounts with deployments that could potentially contain critical user data.
+====
+
+To remove your deployments, please https://account.fusionauth.io/account/deployment/[log into your account and navigate to your deployments]. Once you see your deployment, you can click the [uielement]#Action# button and select [uielement]#Delete#. After a deployment is destroyed, you won't be charged going forward. Any previous usage outside of a free trial will be billed.
```

Comment:
>  you won't be charged going forward.

You won't be charged for the cost associated with this running instance going forward.